### PR TITLE
Add resource requests/limits

### PR DESCRIPTION
**Pull request by vpavlin**
Please, let me know if you don't think these limits are reasonable.

### CODE REVIEW

**Comment by aslakknutsen** [comment]
Hard to tell. We haven't really seen it under any form of load. :)

**Comment by vpavlin** [comment]
Can we do any load testing? Or is that too artificial?

**Comment by vpavlin** [comment]
We cannot got to prod without limits...

File: OpenShiftTemplate.yml [hunk 1]
```diff
@@ -15,6 +15,21 @@ objects:
     selector:
       app: f8tenant
       deploymentconfig: f8tenant
+    strategy:
+      resources:
+        requests:
```
Comment (aslakknutsen) [inline OpenShiftTemplate.yml]: I thought OSO ppl said generally stay away from request. Set limits only?

**Comment by vpavlin** [comment]
@aslakknutsen I've updated this - removed `requests` and lifted the limits a bit. Please review

**Pull request merged**